.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 matrix:
   allow_failures:
-    - php: 7.0
+    - php: hhvm
```

Comment:
i would rather skip those tests on hhvm. and have the plugin throw an exception in the constructor if it is incompatible with hhvm